upload/MultiFile.php

```diff
@@ -1,77 +1,83 @@
 <?php
+
 namespace upload;
 
 class MultiFile
 {
-  public $Files = array();
-  public $Errors = array();
+    public $Files = [];
+    public $Errors = [];
 
-  function __construct($key)
-  {
-    for ($i=0; $i < count($_FILES[$key]['name']); $i++) {
-      if (is_uploaded_file($_FILES[$key]['tmp_name'][$i])) {
-        $data = array(
-          'name' => $_FILES[$key]['name'][$i],
-          'tmp_name' => $_FILES[$key]['tmp_name'][$i]
-          );
-        $this->Files[] = new File($key, $data);
-      }
+    public function __construct($key)
+    {
+        for ($i = 0; $i < count($_FILES[$key]['name']); $i++) {
+            if (is_uploaded_file($_FILES[$key]['tmp_name'][$i])) {
+                $data = [
+          'name'     => $_FILES[$key]['name'][$i],
+          'tmp_name' => $_FILES[$key]['tmp_name'][$i],
+          ];
+                $this->Files[] = new File($key, $data);
+            }
+        }
     }
-  }
 
-  public function Upload($newPath)
-  {
-    $returns = array();
-    if (empty($this->getErrors())) {
-      foreach ($this->Files as $File ) {
-        $returns[] = $File->Upload($newPath);
-      }
-      return $returns;
-    } else {
-      return false;
+    public function Upload($newPath)
+    {
+        $returns = [];
+        if (empty($this->getErrors())) {
+            foreach ($this->Files as $File) {
+                $returns[] = $File->Upload($newPath);
+            }
+
+            return $returns;
+        } else {
+            return false;
+        }
     }
-  }
 
-  public function Size($size)
-  {
-    foreach ($this->Files as $File ) {
-      $File->Size($size);
+    public function Size($size)
+    {
+        foreach ($this->Files as $File) {
+            $File->Size($size);
+        }
+
+        return $this;
     }
-    return $this;
-  }
 
-  public function Extension($AllowedExtensions)
-  {
-    foreach ($this->Files as $File ) {
-      $File->Extension($AllowedExtensions);
+    public function Extension($AllowedExtensions)
+    {
+        foreach ($this->Files as $File) {
+            $File->Extension($AllowedExtensions);
+        }
+
+        return $this;
     }
-    return $this;
-  }
 
-  public function Exist()
-  {
-	  foreach($this->Files as $File) {
-		  $File->Exist();
-	  }
-	  return $this;
-  }
+    public function Exist()
```

Comment:
__[phpmd]__
* The method Exist is not named in camelCase.
* The variable $File is not named in camelCase.

<!-- Comment by [SideCI](https://sideci.com) -->